docs/make.bat

```diff
@@ -0,0 +1,263 @@
+@ECHO OFF

```

Comment:
Do we really need a .bat file? I find it very hard to believe that anyone will be working on the documentation on Windows (or DOS, for that matter).